### PR TITLE
Fix fid import scipy

### PR DESCRIPTION
**Pull request by sdesrozis**
Description:

Fix import error with scipy

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

File: ignite/metrics/gan/fid.py [hunk 1]
```diff
@@ -71,10 +71,6 @@ class FID(_BaseInceptionMetric):
 
         __ https://github.com/mseitzer/pytorch-fid
 
-    .. note::
-        The default Inception model requires the `torchvision` module to be installed.
-        FID also requires `scipy` library for matrix square root calculations.
-
```
Comment (vfdev-5) [inline ignite/metrics/gan/fid.py:77]: @sdesrozis why did you remove this note  ?

File: ignite/metrics/gan/fid.py [hunk 2]
```diff
@@ -129,7 +125,7 @@ def __init__(
 
         if num_features is None and feature_extractor is None:
             num_features = 1000
-            feature_extractor = InceptionModel(return_features=False, device=device)
+            feature_extractor = InceptionModel(return_features=False)
```
Comment (vfdev-5) [inline ignite/metrics/gan/fid.py:128]: @sdesrozis why you also removed device ?

**Comment by sdesrozis** [comment]
Surprising. I will fix this.

**Review by vfdev-5 — APPROVED** [review]
Thanks @sdesrozis
